### ISSUE TITLE
Bump the instance storage 

### ISSUE THREAD

**Issue by gangov** [Phoenix-Protocol-Group/phoenix-nft-marketplace#26]
our contracts currently increase only the persistent storage, but we have to increase the TTL of the instance storage as well, even though we're not explicitly using it to storage any values

**Comment by ueco-jb** [comment]
This also changes after #33 which implemented `ADMIN` as a part of the instance storage.

**Comment by gangov** [comment]
yes, I noticed we don't bump all the storage keys in `collections`. I'll expand on this

**Comment by gangov** [comment]
solved in bc2419a9cac20259ae59056491c9a6b9cdf8e6ae
